src/wizards/tokenSwapDealWizard/stages/tokenDetailsStage/tokenDetailsStage.ts

```diff
@@ -80,14 +80,21 @@ export class TokenDetailsStage {
       this.checkedForUnsavedChanges();
 
       return this.form.validate()
-        .then(async (result) => result.valid &&
-          this.hasValidPrimaryDAOTokensDetailsCount &&
-          !this.hasUnsavedChangesForPrimaryDetails &&
-          !this.hasUnsavedChangesForPartnerDetails &&
-          this.hasValidPartnerDAOTokensDetailsCount &&
-          primaryTokensValid &&
-          (this.isOpenProposalWizard ? true : partnerTokensValid),
-        );
+        .then(async (result) => {
+          if (result.valid) {
+            if (this.wizardState.registrationData.fundingPeriod === undefined) {
+              this.wizardState.registrationData.fundingPeriod = null;
```

Comment:
Before: `delete registrationData.fundingPeriod;`
After: `.fundingPeriod = null`

Can you elaborate?

Reply:
Yes, we set it to `null` in order to fix a bug when resetting fundingPeriod. It was impossible to reset funding period after it was set (for open proposal), for example trying to change from 100 day to nothing (empty input). FundingPeriod in Firestore was remaining untouched. That's why now we save it as `null`

Reply:
You've confirmed that undefined and null behave the same everywhere in the app wrt partnerDAO?